src/com/epam/javatrainings/annotations/CustomerDtoDemo.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.javatrainings.annotations;
+
+
+import com.epam.javatrainings.annotations.exception.EmailNotValidException;
+import com.epam.javatrainings.annotations.exception.InappropriateAgeException;
+import com.epam.javatrainings.annotations.exception.InappropriateNameException;
+import com.epam.javatrainings.annotations.exception.WrongDiscountException;
+
+import java.time.LocalDate;
+
+public class CustomerDtoDemo {
+    public static void main(String[] args) throws NoSuchFieldException, InappropriateNameException, WrongDiscountException, EmailNotValidException, InappropriateAgeException, IllegalAccessException {
+
+        CustomerDto customerDto = new CustomerDto("Anna", "anya.qaramyan00@gmail.ru", LocalDate.of(2000, 7, 12), 30);
+        if(Validation.validate(customerDto)){
```

Comment:
"validate" method should not return boolean, if the method returns boolean, better to call it isValid